ruby/.rubocop.yml

```diff
@@ -1,10 +1,17 @@
+require:
+  - rubocop-rake
```

Comment:
What does `rubocop-rake` does? It does static analysis on the Rakefile?

Reply:
Yes, there are specific rules for rake files, I will remove it

Reply:
Thanks